participants.csv

```diff
@@ -1,2 +1,2 @@
 Zaccharie Ramzi,
-
+Guillaume Daval-Frérot,
```

Comment:
Did you add a new line at the end of the file?

Reply:
No, actually I explicitly removed it. I can add it back if it makes sense.